src/main/java/org/bookstore/service/BookService.java

```diff
@@ -0,0 +1,10 @@
+package org.bookstore.service;
+
+import java.util.List;
+import org.bookstore.model.Book;
+
+public interface BookService {
+    Book save(Book book);
+
+    List<Book> getAll();
```

Comment:
![image](https://github.com/Mariana-VV/BookStoreApp/assets/73041091/f1f938d1-5f77-45f2-915d-5b5c41d7a4e0)

```suggestion
    Book save(Book book);

    List<Book> findAll();
```